lua/telescope/builtin/git.lua

```diff
@@ -113,6 +114,31 @@ git.status = function(opts)
     return
   end
 
+  local displayer = entry_display.create {
+  separator = " ",
+  items = {
+      { width = 2 },
+      { remaining = true },
+    }
+  }
+
+  local make_display = function(entry)
+    local status_highlight
+
+    if entry.status == "??" then
+      status_highlight = "DiffDelete"
+    elseif string.match(entry.status, " .") then
+      status_highlight = "DiffChange"
+    else
+      status_highlight = "DiffAdd"
+    end
```

Comment:
Thats not correct. I would make two 1 width items for status and split it.
Because what about. I have a staged file and do some more changes we will end up with this `MM`.
Now the first one has to be `DiffAdd` and the second one `DiffChange`